src/js/libgeo/GeoOps.js

```diff
@@ -2589,6 +2589,107 @@ geoOps.TransformPolygon.updatePosition = function(el) {
     }));
 };
 
+geoOps.TrComposeTrTr = {};
+geoOps.TrComposeTrTr.kind = "Tr";
+geoOps.TrComposeTrTr.signature = ["Tr", "Tr"];
+geoOps.TrComposeTrTr.updatePosition = function(el) {
+    var a = csgeo.csnames[el.args[0]];
+    var b = csgeo.csnames[el.args[1]];
+    el.matrix = List.normalizeMax(List.productMM(b.matrix, a.matrix));
+    el.dualMatrix = List.normalizeMax(List.productMM(b.dualMatrix, a.dualMatrix));
+};
+
+geoOps._helper.composeMtMt = function(el, m, n) {
```

Comment:
I don't know. It depends on the level of detail. A comment like “extracted coefficients from CAS computation of matrix product” might be all right, but adds pretty little detail. Otherwise I think that the full computation will require too much space to include in a comment. I'd rather save the Sage Jupyter notebook somewhere, and/or the HTML export from that. We could use the code repository for this, or the website, or a dedicated computations repository.